frameworks/elastic/docs/limitations.md

```diff
@@ -15,13 +15,19 @@ The maximum number of deployable nodes is constrained by the DC/OS cluster's res
 
 During deployment and upgrades, the `serial` strategy does not wait for the Elastic service to reach green before proceeding to the next node.
 
+## Toggling TLS requires doing a full-cluster restart
```

Comment:
I would keep the heading as just `Toggling TLS` and then start the section with this as a complete sentence.